### ISSUE TITLE
`<script>` tag incorrect conversion

### ISSUE THREAD

**Issue by guillheu** [lpil/html-lustre-converter#16]
When converting
```html
<script type="module" src="/priv/static/my_project.mjs">const val = 0</script>
```

The resulting Gleam code :
```gleam
  html.html([], [
    html.head([], [
      html.script(
        [
          attribute.src("/priv/static/my_project.mjs"),
          attribute.type_("module"),
        ],
        [html.text("const val = 0")],
      ),
    ]),
    html.body([], []),
  ])
```

... incorrectly uses the function `html.script`

Instead of `html.script([ <attributes> ], [html.text(" <js script content> ")])` it should be `html.script([ <attributes> ], " <js script content> ")`, using the raw script content string instead of an array of `html.text`.

**Comment by lpil** [comment]
Thank you

**Comment by giacomocavalieri** [comment]
This was closed by #18!
